run/authentication/auth.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START run_service_to_service_auth]
+import urllib
+
+import google.auth.transport.requests
+import google.oauth2.id_token
+
+
+def make_get_request(service_url):
```

Comment:
Can we have a more descriptive method name that describes what this code does? A get request sounds like it could be anything.

Maybe `make_authorized_get_request`.